api/data_refinery_api/tests.py

```diff
@@ -629,7 +628,7 @@ def test_processed_samples_only(self):
         )
         self.assertEqual(response.json()["count"], 1)
 
-        qs = Experiment.processed_public_objects
+        Experiment.processed_public_objects
```

Comment:
Bug? Looks like we would have intended to use this.

Reply:
I think we can safetly remove the whole line, since `qs` wasn't used here. And we don't need to test `Experiment.processed_public_objects`